ScratchWikiSkinTemplate.php

```diff
@@ -200,7 +219,7 @@ public function execute() {
 					<div id="feet" style="margin: 0<?=$darkPref ? '' : '; display: none'?>"><?=$line?></div>
 					</div>
 				</div>
-				<ul id="feet">
+				<ul>
```

Comment:
What's the purpose of the change here?

Reply:
A further note: this seems to break the display of the footer links:
![image](https://github.com/InternationalScratchWiki/ScratchWikiSkin2/assets/1960707/114e577b-19b3-44fb-9dc3-e824d75b2532)

This is how it's supposed to look:
![image](https://github.com/InternationalScratchWiki/ScratchWikiSkin2/assets/1960707/30892eee-0bcb-40a6-87ce-816ca536ddd7)

Reply:
Hmm. I was trying to fix there being two elements with the ID "feet" which is invalid. But it seems that the CSS was looking for both of them. I will fix this by converting it into a class.

Reply:
Fixed